Modify verbiage on AWS CLI (#10029)
gravitational/teleport: docs/pages/application-access/guides/aws-console.mdx

- Setup example AWS IAM Read Only and Power User roles.
- Use Teleport's role-based access control with AWS IAM roles.
- View Teleport users' AWS console activity in CloudTrail.
- Access the AWS Command Line Interface (CLI) through Teleport. 

> - 

Reviewer: ```suggestion
```
Accidental extra bullet

  or [Connecting Apps](./connecting-apps.mdx) guides to get it running.
- IAM permissions in the AWS account you want to connect.
- AWS EC2 or other instance where you can assign a IAM Security Role for the Teleport Agent.

> - `aws` Command Line Interface (CLI) tool in PATH. How to [Install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).

Reviewer: ```suggestion
- `aws` command line interface (CLI) tool in PATH. [Installing or updating the latest version of the AWS CLI
](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
```
Changing the link to include the full title of the guide

## Step 8. Using AWS CLI

> Prequisite:

Reviewer: ```suggestion
Before beginning this step, make sure that the `aws` command line interface (CLI) tool is installed in PATH. For more information, read [Installing or updating the latest version of the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
```
If we're adding the prerequisite here, I suggest making it more conversational to fit with the tone of the body text.

## Step 8. Using AWS CLI

Prequisites:

> - `aws` Command Line Interface (CLI) tool in PATH. How to [Install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).

Reviewer: I think this should be in the initial Prerequisites section so a user can do all of their outside-the-guide setup first (as much as possible), then stay in the guide until the end.

Author: I copied this to the top as well. I think good to have both in both places

Author: @ptgott please confirm this is good

Reviewer: @stevenGravy Just added some quick suggestions

Author: Done, thanks @ptgott !

## Step 8. Using AWS CLI

Before beginning this step, make sure that the `aws` command line interface (CLI) tool is installed in PATH. For more information, read [Installing or updating the latest version of the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).

> - `aws` Command Line Interface (CLI) tool in PATH. How to [Install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).

Reviewer: ```suggestion
```
Forgot to mention that this should be removed since we are turning this into a sentence at the beginning of the section. Sorry, @stevenGravy!

Author: np, done @ptgott
